docs/components/Address.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 1
+---
+
+# Address
+
+Display an address (or ENS) plus a copy address icon utility. If the address has an ENS with avatar associated, it will show this avatar, if not it will show a blockie image for that address.
+
+By default, clicking on the address sends you to blockexplorer details for that address.
```

Comment:
Maybe we could mention: that it sends you to [`targetNetwork`](https://scaffold-eth-2-docs-git-update-components-buidlguidldao.vercel.app/deploying/deploy-nextjs-app#--targetnetwork) chain blockexplorer for that address ? 

we can reword / make it smaller or lol skipt it if it gets confusing

Reply:
Thanks a lot for the suggestions! Reviewed the description and changed it a bit, and added the `targetNetwork` part.

**New description for Address:**

Display an address (or ENS) along with a utility icon to copy the address. If the address is associated with an ENS that has an avatar, this avatar will be displayed. If not, a blockie image representation of the address will be shown.

By default, clicking on the address redirects you to the `targetNetwork` blockexplorer with the details of that address.